frontend/base/src/test/java/com/virtuslab/gitmachete/resourcebundles/junit/GitMacheteBundlePropertiesTestSuite.java

```diff
@@ -0,0 +1,35 @@
+package com.virtuslab.gitmachete.resourcebundles.junit;
+
+import static org.junit.Assert.assertTrue;
+
+import java.io.*;
+import java.util.Properties;
+
+import org.junit.Test;
+
+public class GitMacheteBundlePropertiesTestSuite {
+
+  private final String macheteBundleProperties = "GitMacheteBundle.properties";
+
+  @Test
+  public void htmlProperties_should_have_correct_syntax() throws IOException {
+
+    Properties property = new Properties();
+    ClassLoader loader = Thread.currentThread().getContextClassLoader();
+    property.load(loader.getResourceAsStream(macheteBundleProperties));
+
+    property.entrySet().stream()
+        .filter(prop -> ((String) prop.getValue()).contains("<"))
+        .forEach(t -> assertTrue(isCorrectHtmlPropertyFormat((String) t.getKey(), (String) t.getValue())));
```

Comment:
Actually! the error messages from this tests aren't going to be particularly helpful :/ I'd rather create a separate assertion for each of the 3 conditions from line 31, each with a developer-friendly error message

Reply:
stick to the principle of one assertion per test Should I create three tests ? Or break this rule in this case ?

Reply:
I wouldn't stick to that principle too religiously... 3 closely related assertions like these in a single test are okay!